common/src/Vec.h

```diff
@@ -760,6 +785,17 @@ class Vec {
                 return false;
         return true;
     }
+
+    Vec<T,S> snapped(const T precision) const {
+        return Vec<T,S>(*this).snap(precision);
+    }
+
+    Vec<T,S>& snap(const T precision) const {
+        for (size_t i = 0; i < S; ++i) {
+            v[i] = Math::snap(v[i], precision);
```

Comment:
I don't see how this compiles, shouldn't modifying `v` fail since this `snap` function is const?

Reply:
Wow, interesting! No compiler caught this, it seems. Good that we will get rid of these methods soon.